src/tests/UnitTests/Logging/Mocks/DummyLogProvider.cs

```diff
@@ -2,21 +2,18 @@
 
 namespace CodeMonkeys.UnitTests.Logging.Mocks
 {
-    public class DummyLogProvider : LogServiceProvider<DummyLogOptions>
+    public class DummyLogProvider :
+        ILogServiceProvider
     {
-        public DummyLogProvider(DummyLogOptions options) 
-            : base(options)
+        public DummyLogProvider()
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove this redundant constructor.](https://app.codacy.com/gh/UltimateCodeMonkeys/CodeMonkeysMVVM/pullRequest?prid=5645023)